tests/spread/general/core22-step-dependencies/task.yaml

```diff
@@ -15,12 +15,12 @@ execute: |
   fi
 
   # save the time of file left in first pulling
-  TIMESTAMP1=`stat parts/part2/src/testfile --format=%y`
+  TIMESTAMP1=$(stat parts/part2/src/testfile --format=%y)
 
   # pull again, but it should be a noop as part2 already was pulled
   snapcraft pull
```

Comment:
this pull needs to be changed to stage or there needs to be a third step that stages to prevent or detect the regression from happening again